Capstone-Game/Assets/Scripts/AI/Humans.cs

```diff
@@ -146,6 +157,7 @@ public void Update()
                 break;
             }
         }
+        return false;
```

Comment:
You are returning false in a void method. It's causing Unity to load in safe mode.

Reply:
I don't see this? Both my returns falses are in "bool" functions